python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -53,6 +53,7 @@
 """
 
 import re
+import sys
```

Comment:
nit: we can probably remove this line now

Reply:
Removed it, thanks!